sentry/src/db/analytics.rs

```diff
@@ -113,3 +147,86 @@ fn get_time_frame(timeframe: &str) -> (i64, i64) {
         _ => (hour, day),
     }
 }
+
+async fn stat_pair(
+    conn: MultiplexedConnection,
+    key: &str,
+) -> Result<HashMap<String, f64>, Box<dyn Error>> {
+    let data = redis::cmd("ZRANGE")
+        .arg(key)
+        .arg(0 as u64)
+        .arg(-1 as i64)
+        .arg("WITHSCORES")
+        .query_async::<_, Vec<String>>(&mut conn.clone())
+        .await?;
+
+    Ok(data
+        .chunks(2)
+        .map(|chunk: &[String]| {
+            (
+                chunk[0].clone(),
```

Comment:
Are we sure we always have 2 elements? `chunks` can return only 1 element if there is 1 element at the end.

Reply:
yeah we always have even number of elements from Redis if any data is available